### DISCUSSION TITLE
Extra modal footer action in a ViewAction doesn't do anything

### DISCUSSION THREAD

**Discussion by emsifa** [filamentphp/filament#12079] [Help]
### Package

Actions

### Package Version

v3.2.61

### How can we help you?

Hello, I'm trying to add verify button in a `ViewAction` modal, but it doesn't do anything. I also check the browser console, and there is no error.

I follow this documentation: https://filamentphp.com/docs/3.x/actions/modals#opening-another-modal-from-an-extra-footer-action

Here is the code:

```php
->actions([
    // It work
    Tables\Actions\Action::make('verify')->requiresConfirmation()->action(fn () => dd("do verify")),
    Tables\Actions\ViewAction::make()
        ->modalHeading('Registration Detail')
        ->extraModalFooterActions([
            // It doesn't work
            \Filament\Actions\Action::make('verify1')->requiresConfirmation()->action(fn () => dd("do verify")),

            // Still doesn't work
            \Filament\Actions\Action::make('verify2')->action(fn () => dd("do verify")),
        ]),
        Tables\Actions\DeleteAction::make(),
])
```

Here is the demo:

https://github.com/filamentphp/filament/assets/6297931/dc98aa21-1923-4757-8bbe-1df494be57da

Am I missing something?

**Comment by flolanger** [comment]
This seems like the same problem as I experience with https://github.com/filamentphp/filament/discussions/13213

**Comment by s5w95** [comment]
For other people with the same Problem:

dosnt't work with:
use Filament\Actions\Action;

you need to use:
use Filament\Tables\Actions\Action;
